study.md

```diff
@@ -49,19 +50,24 @@ unique. Write the migration would be used to create a table satisfying these
 requirements.
 
 ```ruby
-# your response here
+create_table :pets, id: false do |t|
+  t.string :name, null: false, unique: true
+  t.string :breed
+
+  t.timestamps
+end
 ```
 
 ## Explain the Role of Seed Data
 
 In your own words, explain the role of application seed data.
 
 ```md
-<!-- your response here -->
+Seed is used to add initial data after database is created.
 ```
 
 Should seed data be used for experimentation during development?
 
 ```md
-<!-- your response here -->
+No, because seed is created to add initial data in database. To modify data during development, migrations should be used. 
```

Comment:
Seed data should be used for testing development since it's just temporary data.